requirements.in

```diff
@@ -16,6 +16,7 @@ click==7.1.2              # via black, mozilla-jetstream
 coverage==5.3             # via mozilla-jetstream, pytest-cov
 deprecated==1.2.10        # via pygithub
```

Comment:
looks like this needs to be generated again if we have pygithub deps hanging around